python/love/commander/reports.py

```diff
@@ -27,16 +27,21 @@
 from aiohttp import web
 from astropy.time import Time
 from lsst.ts.m1m3.utils import BumpTestTimes
+from lsst.ts.xml.enums.MTM1M3 import BumpTest as BumpTestStatus
 from lsst.ts.xml.tables.m1m3 import force_actuator_from_id
 
 EFD_CLIENT_CONNECTION_TIMEOUT = 5
 SITE_DOMAINS = {
     "summit_efd": "summit-lsp.lsst.codes",
+    "base_efd": "base-lsp.lsst.codes",
+    "tucson_efd": "tucson-teststand.lsst.codes",
     "usdf_efd": "usdf-rsp.slac.stanford.edu",
 }
 CHRONOGRAF_DASHBOARDS_PATHS = {
     "m1m3_bump_tests": {
         "summit_efd": "/chronograf/sources/1/dashboards/199",
+        "base_efd": "/chronograf/sources/1/dashboards/199",
```

Comment:
https://base-lsp.lsst.codes/chronograf/sources/1/dashboards/71 - now..

Reply:
Thanks Petr! just added it to TTS too: https://tucson-teststand.lsst.codes/chronograf/sources/1/dashboards/47.